tests/integration-tests/tests/common/utils.py

```diff
@@ -121,12 +121,9 @@ def retrieve_pcluster_ami_without_standard_naming(region, os, version, architect
 
 @retry(stop_max_attempt_number=3, wait_fixed=5000)
 def fetch_instance_slots(region, instance_type, multithreading_disabled=False):
-    instance_type_info = get_instance_info(instance_type, region)
-    vcpus = instance_type_info.get("VCpuInfo").get("DefaultVCpus")
-    default_threads_per_core = instance_type_info.get("VCpuInfo").get("DefaultThreadsPerCore")
-    if default_threads_per_core is None:
-        supported_architectures = instance_type_info.get("ProcessorInfo", {}).get("SupportedArchitectures", [])
-        default_threads_per_core = 2 if "x86_64" in supported_architectures else 1
+    vcpu_info = get_instance_info(instance_type, region).get("VCpuInfo")
+    vcpus = vcpu_info.get("DefaultVCpus")
+    default_threads_per_core = vcpu_info.get("DefaultThreadsPerCore")
```

Comment:
if `vcpu_info` doesn't contain one of the two properties (`DefaultVCpus` or `DefaultThreadsPerCore`) can you please verify that the error is clear?

Reply:
After our discuss, change to raise exception when DefaultThreadsPerCore or DefaultVCpus not found

Reply:
Changed